source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -4,6 +4,12 @@ import * as Enums from "./enums";
 import * as Shared from "./shared";
 import { HostConfig } from "./host-config";
 
+export function isMobileOS(): boolean {
+    let userAgent = window.navigator.userAgent;
+
+    return !!userAgent.match(/Android/i) || !!userAgent.match(/iPad/i) || !!userAgent.match(/iPhone/i);
```

Comment:
Are these the only 2 iOS device types we care about? (iPad/iPhone) ?

Reply:
There is only one more device, the iPod, and I don't think we really care.

Reply:
This feels a bit crude of a check. Do we have a desire to behave differently if a user checked on "Show Desktop Version" or any other mobile-browser toggles?

Reply:
Why is it crude? It detects if the renderer is running on a mobile OS, in which case things must be done differently for autoplay to work.